app/templating/summary/question.py

```diff
@@ -1,63 +1,82 @@
         summary_answers = []
         for answer_schema in answer_schemas:
             if 'parent_answer_id' in answer_schema:
                 continue
-            if question_schema['type'] == 'RepeatingAnswer':
-                for answer_value in self._get_answers(answer_store, answer_schema['id']):
-                    answer = self._build_answer(answer_store, question_schema, answer_schema, answer_schemas,
-                                                answer_value)
-                    summary_answers.append(Answer(answer_schema, answer))
             else:
-                answer_value = self._get_answer(answer_store, answer_schema['id'])
-                answer = self._build_answer(answer_store, question_schema, answer_schema, answer_schemas, answer_value)
-                child_answer_value = self._find_other_value_in_answers(answer_store, answer_schema, answer)
-                summary_answers.append(Answer(answer_schema, answer, child_answer_value).serialize())
+                answer_values = self._get_answers(answer_store, answer_schema['id'], group_instance)
+                for answer_value in answer_values:
+                    answer = self._build_answer(answer_store, question_schema, answer_schema, answer_schemas,
+                                                group_instance, answer_value)
+                    child_answer_value = self._find_other_value_in_answers(answer_store, answer_schema, answer, group_instance)
+                    summary_answer = Answer(answer_schema, answer, group_instance, child_answer_value).serialize()
+                    if summary_answer['type'] == 'relationship':
+                        summary_answer['label'] = self._relationship_answer_label(summary_answer['label'], question_schema['parent_id'],
+                                                                                  answer_store, group_instance, question_schema.get('member_label'),
+                                                                                  answer_values.index(answer_value))
+                    summary_answers.append(summary_answer)
+
         return summary_answers
 
-    def _find_other_value_in_answers(self, answer_store, answer_schema, answer):
+    def _relationship_answer_label(self, answer_label, block_id, answer_store, group_instance, member_label, index):
+        group = self.schema.get_group(self.schema.get_block(block_id)['parent_id'])
+        answer_ids = []
+
+        repeat_rule = group['routing_rules'][0]['repeat']
+        if 'answer_ids' in repeat_rule:
+            for answer_id in repeat_rule['answer_ids']:
+                answer_ids.append(answer_id)
+        if 'answer_id' in repeat_rule:
+            answer_ids.append(repeat_rule['answer_id'])
+
+        choice = build_relationship_choices(answer_ids, answer_store, group_instance, member_label)[index]
+        label = answer_label % dict(current_person=choice[0], other_person=choice[1])
```

Comment:
I won't block the PR on this, but is there a reason we're doing string format using reasonably obscure operators?  I had to look up what % did in this context because I'd not come across it before